content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -85,54 +87,68 @@ Download the MSI file for the [Oracle DBM build][4].
 Set `APIKEY` and run the following command in the command prompt inside the directory where you downloaded the installer, for example:
 
 ```shell
-start /wait msiexec /qn /i datadog-agent-7.46.0-dbm-oracle-0.34-1.x86_64.msi APIKEY="" SITE="datadoghq.com"
+start /wait msiexec /qn /i datadog-agent-7.47.0-dbm-oracle-0.40-1.x86_64.msi APIKEY="" SITE="datadoghq.com"
 ```
+[4]: https://s3.amazonaws.com/ddagent-windows-stable/beta/datadog-agent-7.47.0-dbm-oracle-0.40-1.x86_64.msi
+[8]: https://ddagent-windows-stable.s3.amazonaws.com/
 
-##### Docker
-
-The docker Oracle DBM images can be found [here][9].
+{{% /tab %}}
+{{% tab "Docker" %}}
+The Oracle DBM images for Docker can be found [here][9].
```

Comment:
@caseycull I realized that we're not supposed to use "here" links, and this should probably be a clearer instruction anyway. When we link these build repositories, are we asking them to download the build they need, then run the shell command in the folder with the downloaded build? Same question for all of these instances that say something similar about a build repository.